utilities.tex

```diff
@@ -785,28 +792,28 @@
     tagged(const tagged&) = default;
     tagged &operator=(tagged&&) = default;
     tagged &operator=(const tagged&) = default;
-    tagged(Base&&) noexcept(@\seebelow@)
+    constexpr tagged(Base&&) noexcept(@\seebelow@)
       requires MoveConstructible<Base>;
-    tagged(const Base&) noexcept(@\seebelow@)
+    constexpr tagged(const Base&) noexcept(@\seebelow@)
```

Comment:
You thought you could sneak these by me, didn't you? ;-) P0579 doesn't specify these two to be `constexpr`. These constructors came from the resolution of #172, which also doesn't specify them to be `constexpr`. If we're playing by the book, we should leave the `constexpr` off here and file an issue. I'm content to look the other way if you wanted to leave them in, though.